README.md

```diff
@@ -2,74 +2,138 @@
 Bowling Challenge
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+## Goals
+This is the week 6 weekend challenge for Makers Academy. During this week, we were introduced to Javascript and JQuery.
 
-## The Task
+- Translate last weekend's bowling challenge from Ruby into Javascript.
+- Use JQuery to create a user interface.
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+## My Project
 
-Count and sum the scores of a bowling game for one player (in JavaScript).
+## Set Up
+Clone the repository:
+```
+$ git clone https://github.com/emilyalice2708/bowling-challenge-ruby
```

Comment:
Not ruby anymore!

Reply:
Haha don't know how I missed that! Thank you!